callbacks.py

```diff
@@ -62,13 +63,83 @@ def get_prediction_masks(self):
             outputs_batch = self.model(X)
             if len(outputs_batch) == len(self.output_names):
                 for name, output, target in zip(self.output_names, outputs_batch, targets_tensors):
+                    if name not in self.outputs_to_plot:
+                        continue
```

Comment:
@apyskir I see that you use if ...: continue a lot. It is of course ok to use but I feel that it is often more readable if you simply go if something: do this else: do that. The flow of execution is easier to follow.

Reply:
OK